aten/src/ATen/native/native_functions.yaml

```diff
@@ -3792,6 +3792,8 @@
     CPU: mkldnn_rnn_layer_backward
   autogen: mkldnn_rnn_layer_backward.out
 
+- func: mkldnn_rnn_layer_differentiable_backward(Tensor input, Tensor weight1, Tensor weight2, Tensor weight3, Tensor weight4, Tensor hx_, Tensor cx_tmp, Tensor output, Tensor hy_, Tensor cy_, Tensor? grad_output, Tensor? grad_hy, Tensor? grad_cy, bool reverse, int mode, int hidden_size, int num_layers, bool has_biases, bool train, bool bidirectional, int[] batch_sizes, bool batch_first, Tensor workspace) -> (Tensor, Tensor, Tensor, Tensor, Tensor, Tensor, Tensor)
```

Comment:
Why does this need to be a native function? Could we put this in FunctionsManual instead?

Reply:
This is to imitate `_thnn_differentiable_lstm_cell_backward` op.

You mean, here we need to put the op implementation into FunctionsManual.cpp, and remove it from `native_functions.yaml`, so that the op can be used in `derivatives.yaml`?

Reply:
> You mean, here we need to put the op implementation into FunctionsManual.cpp, and remove it from native_functions.yaml, so that the op can be used in derivatives.yaml?

Yeah, having an entry `native_functions.yaml` seems unnecessary. For a function to be used in `derivatives.yaml` its name just needs to be in the native namespace, e.g. putting it in `FunctionsManual.cpp` should be enough.

Reply:
Done. Thanks for the comments.